examples/geospatial.rs

```diff
@@ -0,0 +1,62 @@
+extern crate redis;
+
+use redis::{geo, Commands, RedisResult};
+use std::env;
+use std::f64;
+use std::process::exit;
+
+fn redis_url() -> String {
+    match env::var("REDIS_URL") {
+        Ok(url) => url,
+        Err(..) => "redis://127.0.0.1/".to_string(),
+    }
+}
+
+fn run() -> RedisResult<()> {
```

Comment:
When we add a feature to conditionnaly disable the geo functionality, we also need to stub this out:

```rust
#[cfg(feature = "geo")]
fn run() -> RedisResult<()> {
 ...
}
#[cfg(not(feature = "geo"))]
fn run() -> RedisResult<()> { Ok(()) }
```

Reply:
Scrap that. We only need to disable the test that is run, not any actual implementation code.